cockroachdb/templates/statefulset.yaml

```diff
@@ -294,6 +303,8 @@ spec:
                 - ALL
             privileged: false
             readOnlyRootFilesystem: true
+            seccompProfile:
+              type: "RuntimeDefault"
```

Comment:
It is not needed here. Already added in pod securityContext

Reply:
oops, sorry about that!